conversion/conversion.go

```diff
@@ -506,19 +506,33 @@ func CreateDatabase(ctx context.Context, adminClient *database.DatabaseAdminClie
 	// Spanner DDL doesn't accept them), and protects table and col names
 	// using backticks (to avoid any issues with Spanner reserved words).
 	// Foreign Keys are set to false since we create them post data migration.
-	schema := conv.SpSchema.GetDDL(ddl.Config{Comments: false, ProtectIds: true, Tables: true, ForeignKeys: false})
-	op, err := adminClient.CreateDatabase(ctx, &adminpb.CreateDatabaseRequest{
+	req := &adminpb.CreateDatabaseRequest{
 		Parent:          fmt.Sprintf("projects/%s/instances/%s", project, instance),
-		CreateStatement: "CREATE DATABASE `" + dbName + "`",
-		ExtraStatements: schema,
-	})
+	}
+	if conv.TargetDb == TARGET_EXPERIMENTAL_POSTGRES {
+		// PG doesn't allow backticks around the database name.
+		req.CreateStatement = "CREATE DATABASE " + dbName
+		req.DatabaseDialect = adminpb.DatabaseDialect_POSTGRESQL
+	} else {
+		req.CreateStatement = "CREATE DATABASE `" + dbName + "`"
+		// PG doesn't allow extra statements during database creation.
```

Comment:
Cut-and-paste duplicate comment?
The one at line 513 seems appropriate, but this one seems out of place?